sklearn/naive_bayes.py

```diff
@@ -115,6 +115,12 @@ class GaussianNB(BaseNB):
 
     Read more in the :ref:`User Guide <gaussian_naive_bayes>`.
 
+    Parameters
+    ----------
+    class_prior_override : array-like, size (n_classes,)
```

Comment:
I think the names should be `class_prior` to be consistent with `BernouilliNB` and `MultinomialNB`